src/api/blogService.js

```diff
@@ -69,8 +69,9 @@ export async function createBlog(data) {
 
 export async function updateBlog({ blogId, data }) {
   data.tags = JSON.stringify(createTagsArr(data.tags));
+  data = createFormData(data);
   try {
-    return await axios.put(`/api/restify/blogs/${blogId}`, data);
+    return await axios.post(`/api/restify/blogs/${blogId}`, data);
```

Comment:
is there a reason why this changed?

Reply:
da, pentru ca nu putem face put cu form-data la edit pentru a edita cover image